doc/sphinx/proof-engine/proof-handling.rst

```diff
@@ -1,581 +1,584 @@
 
 This command is intended to be used to instantiate existential
 variables when the proof is completed but some uninstantiated
 existential variables remain. To instantiate existential variables
-during proof edition, you should use the tactic {\tt instantiate}.
+during proof edition, you should use the tactic instantiate.
+
 
-\SeeAlso {\tt instantiate (\num:= \term).} in Section~\ref{instantiate}.
-\SeeAlso {\tt Grab Existential Variables.} below.
+See also: ``instantiate (num:= term).`` in Section
+:ref:`TODO-controllingtheproofflow`.
+See also: ``Grab Existential Variables.`` below.
 
-\subsection[\tt Grab Existential Variables.]{\tt Grab Existential Variables.\comindex{Grab Existential Variables}
-\label{GrabEvars}}
 
-This command can be run when a proof has no more goal to be solved but has remaining
-uninstantiated existential variables. It takes every uninstantiated existential variable
-and turns it into a goal.
+.. cmd:: Grab Existential Variables.
 
-%%%%%%%%
-\section{Navigation in the proof tree}
-%%%%%%%%
+This command can be run when a proof has no more goal to be solved but
+has remaining uninstantiated existential variables. It takes every
+uninstantiated existential variable and turns it into a goal.
 
-\subsection[\tt Undo.]{\tt Undo.\comindex{Undo}}
 
-This command cancels the effect of the last command.  Thus, it
+Navigation in the proof tree
+--------------------------------
+
+
+.. cmd:: Undo.
+
+This command cancels the effect of the last command. Thus, it
 backtracks one step.
 
-\begin{Variants}
 
-\item {\tt Undo {\num}.}
+.. cmdv:: Undo @num.
 
-  Repeats {\tt Undo} {\num} times.
+Repeats Undo :n:`@num` times.
 
-\end{Variants}
+.. cmdv:: Restart.
 
-\subsection[\tt Restart.]{\tt Restart.\comindex{Restart}}
 This command restores the proof editing process to the original goal.
 
-\begin{ErrMsgs}
-\item \errindex{No focused proof to restart}
-\end{ErrMsgs}
 
-\subsection[\tt Focus.]{\tt Focus.\comindex{Focus}}
-This focuses the attention on the first subgoal to prove and the printing
-of the other subgoals is suspended until the focused subgoal is
-solved or unfocused. This is useful when there are many current
+.. exn:: No focused proof to restart
+
+
+.. cmd:: Focus.
+
+This focuses the attention on the first subgoal to prove and the
+printing of the other subgoals is suspended until the focused subgoal
+is solved or unfocused. This is useful when there are many current
 subgoals which clutter your screen.
 
-\begin{Variant}
-\item {\tt Focus {\num}.}\\
-This focuses the attention on the $\num^{th}$ subgoal to prove.
-\end{Variant}
 
-\emph{This command is deprecated since 8.8: prefer the use of bullets or
-  focusing brackets instead, including {\tt {\num}: \{}}.
+.. cmdv:: Focus @num.
+
+This focuses the attention on the :n:`@num` th subgoal to
+prove.
+
+This command is deprecated since 8.8: prefer the use of bullets or
+focusing brackets instead, including :n:`@num : %{`
```

Comment:
Emphasis was lost. Furthermore, it is less clear now that it is the command and its variant which are deprecated. So what about putting some plural here?

Reply:
Why using stars doesn't work?

Some plural: "This command is" -> "These commands are"

Reply:
I can make it stop before the `:n:...`, but other wise it seems to break it.

Reply:
I see. You can also put just the first part of the sentence (till the colon `:`) in italic.